### DISCUSSION TITLE
Change OCP curve and get an error

### DISCUSSION THREAD

**Discussion by xyyrly** [pybamm-team/PyBaMM#3269] [Q&A] [answered]
Hi team,

I am trying to use another fitting curve of OCP vs sto, but it raises an error: 
Events ['Maximum voltage [V]'] are non-positive at initial conditions. 
Any suggestions? Thanks a lot!

**Comment by brosaplanella** [comment, marked as answer]
This means that the initial concentration gives a voltage higher than the upper cut-off voltage. Either change the cut-off voltage or the initial concentrations in the parameters.

**Reply by xyyrly** [reply]
Thank you!
